tornado-api/src/main/java/uk/ac/manchester/tornado/api/types/arrays/ByteArray.java

```diff
@@ -198,6 +201,16 @@ public void set(int index, byte value) {
         segment.setAtIndex(JAVA_BYTE, baseIndex + index, value);
     }
 
+    public void setHalf(int byteIndex, HalfFloat value) {
```

Comment:
rename setHalfFloat for consistance with the Tornado type for `HalfFloat` as half is not a JVM type